brainreg_segment/tracks/layers.py

```diff
@@ -26,3 +25,10 @@ def add_existing_track_layers(viewer, track_file, point_size):
     )
     new_points_layer.mode = "ADD"
     return new_points_layer
+
+
+def add_track_from_existing_layer(selected_layer, track_layers):
+    """
+    Adds an existing tracks layer (e.g. from another plugin) to the list
+    """
+    track_layers.append(selected_layer)
```

Comment:
Similar to above, I think worth checking the layer type and erroring if not the right error type here.